gmso/tests/test_conversions.py

```diff
@@ -76,3 +111,159 @@ def test_kcal_per_mol_to_string_m(self):
                 input_value,
                 "m",
             )
+
+    def test_conversion_for_topology_dihedrals(self, typed_ethane):
+        expected_units_dim = dict(
+            zip(["c0", "c1", "c2", "c3", "c4", "c5"], ["energy"] * 6)
+        )
+        base_units = u.UnitSystem("atomic", "Å", "mp", "fs", "nK", "rad")
```

Comment:
Will a user ever need to type the symbol, Å? if so, using this nonstandard character could be a concern.